core/prototypes/list.py

```diff
@@ -37,45 +29,57 @@ def adminInfo(self):
 	def __init__(self, moduleName, modulePath, *args, **kwargs):
 		super(List, self).__init__(moduleName, modulePath, *args, **kwargs)
 
-	def viewSkel(self, *args, **kwargs):
+	def viewSkel(self, *args, **kwargs) -> SkeletonInstance:
 		"""
-		Retrieve a new instance of a :class:`server.skeleton.Skeleton` that is used by the application
+		Retrieve a new instance of a :class:`viur.core.skeleton.Skeleton` that is used by the application
```

Comment:
The return type-annotations says `SkeletonInstance`, the docstring says (instance of) `viur.core.skeleton.Skeleton`. Which one is correct?

Reply:
Fixed. SkeletonInstance is correct (instancing a skeleton-class now gives you an instance of SkeletonInstance and never an instance of that skeleton-class anymore)